static/app/components/events/groupingInfo/index.tsx

```diff
@@ -209,11 +209,7 @@ class GroupingInfo extends DeprecatedAsyncComponent<Props, State> {
       <EventDataSection
         type="grouping-info"
         title={t('Event Grouping Information')}
-        actions={
-          <ToggleButton onClick={this.toggle} priority="link">
-            {isOpen ? t('Hide Details') : t('Show Details')}
-          </ToggleButton>
-        }
+        actions={<ToggleButton isShown={isOpen} setShownState={this.toggle} />}
```

Comment:
I'm assuming the `isOpen` state is stored across sessions?

Reply:
that's what replay team is trying to push for (and what we're discussing at the meeting tomorrow). currently, the behavior isn't stored (and never has been)